### ISSUE TITLE
Failure because of "hard coding" generated/random node names 

### ISSUE THREAD

**Issue by plyoung** [misoe92/AssetSnap-Godot#35]
I see you depend on finding control nodes via names like this `@VBoxContainer@14/@HSplitContainer@17/@HSplitContainer@25/@VSplitContainer@27`

It fails in Godot 4.3 dev5 so you might want to start looking into a beter way of finding the nodes.

```
Godot Engine v4.3.dev5.mono.official (c) 2007-present Juan Linietsky, Ariel Manzur & Godot Contributors.
  Node not found: "@VBoxContainer@14/@HSplitContainer@17/@HSplitContainer@25/@VSplitContainer@27" (relative to "/root/@EditorNode@16776/@Panel@13").
  /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs:113 - System.NullReferenceException: Object reference not set to an instance of an object.
     at AssetSnap.Front.Nodes.AsContextMenu._Ready() in C:\Work\game\addons\assetsnap\nodes\interface\AsContextMenu.cs:line 70
     at Godot.Node.InvokeGodotClassMethod(godot_string_name& method, NativeVariantPtrArgs args, godot_variant& ret) in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Generated/GodotObjects/Node.cs:line 2263
     at Godot.CanvasItem.InvokeGodotClassMethod(godot_string_name& method, NativeVariantPtrArgs args, godot_variant& ret) in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Generated/GodotObjects/CanvasItem.cs:line 1388
     at Godot.Control.InvokeGodotClassMethod(godot_string_name& method, NativeVariantPtrArgs args, godot_variant& ret) in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Generated/GodotObjects/Control.cs:line 2842
     at AssetSnap.Front.Nodes.AsContextMenu.InvokeGodotClassMethod(godot_string_name& method, NativeVariantPtrArgs args, godot_variant& ret) in C:\Work\game\Godot.SourceGenerators\Godot.SourceGenerators.ScriptMethodsGenerator\AssetSnap.Front.Nodes.AsContextMenu_ScriptMethods.generated.cs:line 380
     at Godot.Bridge.CSharpInstanceBridge.Call(IntPtr godotObjectGCHandle, godot_string_name* method, godot_variant** args, Int32 argCount, godot_variant_call_error* refCallError, godot_variant* ret) in /root/godot/modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/CSharpInstanceBridge.cs:line 24
  Directory not found: res://groups/
```

**Comment by misoe92** [comment]
Thank you for letting me know :) 

I will look into that and try to get it fixed for when 0.1.2 is ready :)

**Comment by misoe92** [comment]
A fix should be merged into 0.1.2 now, but 0.1.2 is still expiremental since alot of changed has been made to the overall structure of traits to ensure that things are freed as they should.

Ill give a heads up when it gets 100% stable again, prob. in like 3-4 days when im done going through most of the code and updating the relations and such in various of areas that might have broken.
